app/assets/javascripts/application.js

```diff
@@ -64,6 +139,8 @@ function navigateDate(val) {
 }
 
 function dateOnPageLoad(passed_date) {
+    //are we using this for something?
+    debugger;
```

Comment:
Haha! Of course not. But the fact that all my tests are passing WITH THAT in, worries me. ;-)